src/styleConfig.ts

```diff
@@ -24,6 +24,7 @@ export interface StyleConfig {
    *   },
    * });
    * ```
+   * @platform iOS, Android
```

Comment:
How is the difference between iOS and tvOS handled here? 🤔

Reply:
@stonko1994 AFAIK it would show up on the documentation (once we generate it). Not sure about IDEs.
It could easily happen that tvOS is included in iOS. I couldn't find any resource saying one or the other.